lambdas/indexer/app.py

```diff
@@ -1,16 +1,10 @@
 """
 Chalice application module to receive and process DSS event notifications.
 """
-from collections import (
-    Counter,
-    defaultdict,
-)
-from datetime import datetime
+from collections import defaultdict
```

Comment:
Once an import statement is multi line we don't go back to single line, even if just one imported symbol remains. You couldn't have known that but the PyCharm formatter settings at the project root define that. Be sure to import it. If you don't use PyCharm you'll have to enforce those settings manually.